website/django/data/views.py

```diff
@@ -105,6 +108,10 @@ def index(request):
     if search_term:
         query, synonyms_count = apply_search(query, search_term, quotes=quotes, release=release)
 
+    if not show_deleted and not release:
```

Comment:
Currently, they are viewable like other change types, but first thing you see is a blank table and the "N deleted variants match" text. I mentioned this to Melissa and Mary and they said it was low priority, but it should be a quick fix.